packages/ui/build/rollup/configs/esm.js

```diff
@@ -41,7 +41,7 @@ export function createESMConfig ({ input, outDir = 'dist/', minify = false, decl
         compileTemplate: false,
         preprocessStyles: true,
       }),
-      commonjsPlugin(),
+      commonjsPlugin({ include: ['tslib'] }),
```

Comment:
Using filter include marks all other modules to exclude. Because of that, it's impossible to bundle other commonjs modules, that are not in include.